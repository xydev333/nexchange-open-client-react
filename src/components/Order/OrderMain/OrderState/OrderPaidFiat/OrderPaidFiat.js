import React from 'react';
import OrderStateLoader from '../OrderIcons/OrderStateLoader/OrderStateLoader';
import styles from '../OrderState.scss';

const OrderPaid = () => (
  <div className={styles.container}>
    <OrderStateLoader />
    <h2 className={styles.title}>Payment & verification received</h2>
    <h3 className={styles.subtitle}>We are now preparing to release your coins</h3>
  </div>
);

export default OrderPaid;
