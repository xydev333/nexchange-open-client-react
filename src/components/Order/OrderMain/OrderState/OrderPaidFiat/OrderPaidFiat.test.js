import React from 'react';
import { shallow } from 'enzyme';
import OrderPaidFiat from './OrderPaidFiat';

describe('OrderPaidFiat', () => {
  it('renders correctly', () => {
    expect(shallow(<OrderPaidFiat />)).toMatchSnapshot();
  });
});
