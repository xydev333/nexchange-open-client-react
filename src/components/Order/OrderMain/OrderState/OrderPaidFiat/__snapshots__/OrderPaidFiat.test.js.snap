// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderPaidFiat renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderPaid />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <OrderStateLoader />,
        <h2
          className="title"
>
          Payment & verification received
</h2>,
        <h3
          className="subtitle"
>
          We are now preparing to release your coins
</h3>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Payment & verification received",
          "className": "title",
        },
        "ref": null,
        "rendered": "Payment & verification received",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "We are now preparing to release your coins",
          "className": "subtitle",
        },
        "ref": null,
        "rendered": "We are now preparing to release your coins",
        "type": "h3",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <OrderStateLoader />,
          <h2
            className="title"
>
            Payment & verification received
</h2>,
          <h3
            className="subtitle"
>
            We are now preparing to release your coins
</h3>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Payment & verification received",
            "className": "title",
          },
          "ref": null,
          "rendered": "Payment & verification received",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "We are now preparing to release your coins",
            "className": "subtitle",
          },
          "ref": null,
          "rendered": "We are now preparing to release your coins",
          "type": "h3",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
